report/2.Analysis/ProblemStatement.tex

```diff
@@ -9,7 +9,7 @@ \subsection{Naming}
 \subsection{Analysis takeaways}
 The primary takeaways from the analysis helped specify the specific platform and sensors that will be needed in the development of a solution, as well as reasonings for the usage of machine learning and the principles of real-time systems.
 
-The best platform choice was determined to be the NXT, primarily because of CPU limitations, which makes the project more interesting for the group, but also because of its interactability with hardware components.
+The best platform choice was determined to be the NXT, primarily because of CPU limitations, which makes the project more interesting for the group, but also because of its intractability with hardware components.
```

Comment:
intractable
[in-trak-tuh-buhl]
adjective
not easily controlled or directed; not docile or manageable; stubborn; obstinate:

Reply:
Kan ikke lave suggestions men det skal altså være: interactability